pySDC/projects/parallelSDC/environment.yml

```diff
@@ -0,0 +1,10 @@
+name: pySDC
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - mpi4py
+  - numpy
+  - matplotlib>=3.0
+  - dill>=0.2.6
+  - scipy>=0.17.1
```

Comment:
Let's put exactly the base environment here and see if the test times change.

Reply:
I just gave it a try with an environment that is similar (not the same) to the mpi4py-environment. The local results looked promising. Let's see how CI will behave.

Reply:
As the jobs for parallelSDC need only 10-20 minutes (the first one and the reloaded one respectively), I assume, we can close this conversation. If you think otherwise, please reopen it or open a new one.